ansible/configs/three-tier-app/env_vars.yml

```diff
@@ -99,15 +99,23 @@ appdb_instance_count: 1
 support_instance_count: 1
 rootfs_size_bastion: 50
 
+bastion_instance_image: RHELAMI
+frontend_instance_image: RHELAMI
+app_instance_image: RHELAMI
+appdb_instance_image: RHELAMI
+support_instance_image: RHELAMI
```

Comment:
These seem very specific to EC2. Are people expected to change these vars either here or in agnosticV? If so, can we add some comments to explain why they are set like this?

Reply:
Yes they are expected to change it.

We can provide meaning full examples (but idk if openstack comes with images by default so it may just be specific anyway).

see `sample_vars_osp.yml`